### PR TITLE
Update main repo readme

### PR DESCRIPTION
**Pull request by stephhazlitt**
- condenses main repo readme, adds project state and licence badges
- removes unused `bcgovr` template scripts

### CODE REVIEW

**Comment by stephhazlitt** [comment]
A quick attempt at a tighter main repo readme. I suggest a repo reorg/tidy to put the first R attempt work into a folder, with it's own readme re: what are the scripts, what software do you need etc). This main readme should focus on how to get started to run the MVP, e.g. what tools, where to get them etc, how to use the scripts (or direct users to a subfolder readme if there).

**Review by jonjunduan — APPROVED** [review]
It looks great.

**Comment by lindsay-fredrick** [comment]
Looks good to me - just noting that the main readme will need updating with a note if/when the R work all gets moved over to another folder.

**Review by lindsay-fredrick — APPROVED** [review]

**Comment by stephhazlitt** [comment]
Yep. I was thinking of waiting for @jonjunduan to merge those changes into the `dbt` branch and then I will pull those changes into this branch before merging.

**Comment by lindsay-fredrick** [comment]
@stephhazlitt you should be good to merge the changes into your branch now. Assuming that works for you, this should be good to merge back into dbt.

**Review by jonjunduan — APPROVED** [review]
It looks great.

File: README.md
Comment (lindsay-fredrick) [inline README.md:16]:
```suggestion
Some initial prototyping and performance testing was implemented using R&mdash;scripts and documentation of this approach are including in the `/r-implementation` folder. See `/r-implementation/README.md` for details. 
```

File: sei_dbt/README.md
Comment (lindsay-fredrick) [inline sei_dbt/README.md:94]: I thought you needed this footer for every readme?

**Review by lindsay-fredrick — APPROVED** [review]
Left a few small comments, but otherwise looks cleaner now :)

File: sei_dbt/README.md
Comment (stephhazlitt) [reply sei_dbt/README.md:94]: My understanding is that it is needed in the repository `README.md` only, although the guidance [here](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Licenses.md) does not really address this question.

Comment (lindsay-fredrick) [reply sei_dbt/README.md:94]: Yeah that aligns with their [cheatsheet](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md) as well - I think we're all good to remove it.

**Pull request merged**
